dali/test/example_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "dali_operator_test.h"
+
+namespace dali {
+
+namespace testing {
+
+class MyOperatorTest : public DaliOperatorTest<GPUBackend, CPUBackend> {
+ protected:
+  OperatorsGraph GenerateOperatorsGraph() const noexcept override {
+    return "MyOp";
+  };
+
+  std::vector<DataType<GPUBackend>> inputs;
```

Comment:
Saw two variables in a class
 that some data had to pass.
Searched where they get written to
           but alas!

Reply:
You'd like to have everything handled,
With me you rankle,
While this file is just an example...